src/lib/bitwarden.js

```diff
@@ -92,27 +92,29 @@ export function buildCipherDocument(body, user) {
     fields: [],
   };
 
-  let additionalParams = null;
+  let additionalParamsType = null;
   if (params.type === TYPE_LOGIN) {
-    additionalParams = 'login';
+    additionalParamsType = 'login';
   } else if (params.type === TYPE_CARD) {
-    additionalParams = 'card';
+    additionalParamsType = 'card';
   } else if (params.type === TYPE_IDENTITY) {
-    additionalParams = 'identity';
+    additionalParamsType = 'identity';
   } else if (params.type === TYPE_NOTE) {
-    additionalParams = 'securenote';
+    additionalParamsType = 'securenote';
   }
 
-  if (additionalParams && body[additionalParams]) {
-    params[additionalParams] = {};
-    entries(body[additionalParams]).forEach(([key, value]) => {
-      let paramValue = value;
-      if (ucfirst(key) === 'Uris') {
-        paramValue = value.map(val => mapKeys(val, (uriValue, uriKey) => ucfirst(uriKey)));
-      }
-
-      params[additionalParams][ucfirst(key)] = paramValue;
-    });
+  if (additionalParamsType !== null) {
+    const additionalParams = body[additionalParamsType] || body[ucfirst(additionalParamsType)];
```

Comment:
I think this fallback with `ucfirst` is unnecessary, bitwarden sends the body with different cases in the case of every device, so i just lowercase the params at the very beginning of processing the request.

in this case:
https://github.com/vvondra/bitwarden-serverless/blob/master/src/ciphers.js#L14
https://github.com/vvondra/bitwarden-serverless/blob/master/src/ciphers.js#L30

Reply:
You are correct.  This has been fixed.